support_for_advertising_use_cases.md

```diff
@@ -165,19 +190,87 @@ These use-cases are more focused on the publisher's perspective.
+
+## Invalid Traffic
+
+Advertisers want to show ads to humans, not to bots and scripts. The industry has devoted a great deal of time and energy towards identifying invalid traffic (IVT), so that it can be filtered out of impression reports.
+
+## Third-Party Verification
+
+When a measurement vendor (ad server, publisher, etc) reports a certain number of impressions (presumably viewable impressions, with invalid traffic filtered out), how is the advertiser to know if this number is accurate? Advertisers would prefer not to just accept these numbers on faith. For this reason, the industry has established a number of independent, third-party measurement vendors that run their own verification scripts to provide advertisers with the peace of mind and confidence that the number of impressions they are being billed for are accurate, and measured in a consistent way across all of their ad buys.
+
+# Audience Verification
 
+Not all products are suited for all consumers. For example, in the United States sales of alcohol are restricted to 
+people aged 21 and over, and there are even regulations limiting the extent to which advertisements for alcohol can be 
+shown to people under 21. Thus advertisers of alcohol have not only an economic interest but also a legal obligation
+to avoid advertising to teenagers.
+
+The current ecosystem has developed elaborate mechanisms to allow advertisers to reach consumers they believe will have the interest and ability to purchase their products. While the move to a more private web will (and should) eliminate some of the current privacy-unfriendly techniques marketers use for targeting, it will not eliminate the reasonable desire (and in some cases obligation) to avoid showing irrelevant ads to uninterested or inappropriate consumers.  
+
+There is a real cost in assembling and assessing addressable audiences who are united by some common characteristic of 
+interest. As such, vendors who amass these audiences (whether those vendors are publishers or third parties) often charge a substantial premium to advertisers who wish to access these audiences.
+With so much money at stake, advertisers have a strong interest in knowing whether they're actually reaching the types of people they're paying to reach. As targeting moves behind opaque mechanisms like TURTLEDOVE, there is a real risk that without the possibility of independent verification, advertisers lose so much trust in the accuracy of audience targeting mechanisms that they revert to the inefficient and annoying-to-consumers spray-and-pray tactics of the mass media era. This will both hurt consumers who will be bombarded with irrelevant or lowest-common-denominator ads, and it will hurt small publishers for whom providing advertisers access to a niche audience may be their primary economic value proposition. 
```

Comment:
I would prefer to keep this document focused on just explaining various advertiser use-cases.
Some of the language used in this section is commentary on TURTLEDOVE (which belongs in the section of support for these use-cases, but not in the description of the use-case itself).
The speculation about the future of the web also feels out of place.
Could I ask you to remove the emotive language and keep this section a dispassionate description of simply the advertiser's goal / objective / use-case?

Reply:
I am not the author of this paragraph. I only copy-pasted them.
This was commited by @gjlondon in commit d222cbca205b96371bb2f4e419edcf21898e797a